docs/DeveloperGuide.adoc

```diff
@@ -144,11 +289,58 @@ The `Storage` component,
 * can save `UserPref` objects in json format and read it back.
 * can save the Address Book data in json format and read it back.
 
+[[Design-Notification]]
+=== Notification System
+(Picture: Notification System)
```

Comment:
Your group might want to consider putting in a Sequence Diagram to more clearly illustrate the interactions between Daemon and the Main Thread.